style.css

```diff
@@ -470,3 +482,376 @@ h2 {
   width: 134px;
   border: 5px solid #212121;
 }
+
+@media screen and (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  body {
+    overflow-x: hidden;
+  }
+
+  .links {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    gap: 24px;
+  }
+
+  .dropdown {
+    display: flex;
+    flex-direction: row;
+    margin-right: 156px;
+    padding-bottom: 30px;
+  }
+
+  a {
+    text-decoration: none;
+    color: #fff;
+    width: 63px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .mail {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-image: url(./Images/Icon\ -\ Mail.png);
+    margin-left: 20px;
+    margin-top: 20px;
+    gap: 30px;
+  }
+
+  header {
+    width: 100%;
+    height: 8px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .headline {
+    width: 100%;
+    height: 550px;
+    background-image: url(/Images/Illustration-header@2x.png);
+    background-size: 500px 768px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
```

Comment:
### For your headline-container 
- [ ] You did most of the changes. But as mentioned by the previous reviewer, notice your heading text still not using the right `front-family`, in this case `'Crete Round'`, Please make sure your font is getting applied before the next review request.   🤚